src/index.ts

```diff
@@ -9,21 +9,27 @@ import {
 } from '@tesseractcollective/serverless-toolbox';
 
 import EventRouter from './EventRouter';
-import HasuraUserApi, { HasuraUser } from './HasuraUserApi';
-import JwtHasuraAuth, { UserPassword } from './JwtAuth';
-import HasuraAuthRouter from './AuthRouter';
+import JwtAuth, { UserPassword } from './JwtAuth';
+import AuthRouter from './AuthRouter';
+
+// HasuraUserApi is only used if env vars are set
+import HasuraUserApi from './HasuraUserApi';
+
+const hasuraUrl = process.env['HASURA_URL'];
+const hasuraAdminSecret = process.env['HASURA_ADMIN_SECRET'];
```

Comment:
Do we want to use getEnvVar() here?

Reply:
No. `getEnvVar()` throws an error if it's not there. Since this one is optional, I'm doing it this way.